src/screens/SignIn/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Redirect } from 'react-router-dom'
+
+import SignInForm from 'screens/SignIn/form'
+import SubmitHandler from 'screens/SignIn/handler'
+
+const SignIn = (): JSX.Element => {
+  const handler = SubmitHandler()
```

Comment:
can we destructure this rather than using `handler.requestSuccess`? 
`{ requestSuccess, error, handleSubmit } = SubmitHandler()`

Reply:
Fixed in 8dfa4e76d9b6c6d824e4113a785b34db9c4f514a